api/api.go

```diff
@@ -76,6 +76,10 @@ func request(method string, u url.URL, body io.Reader, headers ...http.Header) (
 
 // prettyTxIDError creates a support friendly error message with an transaction ID
 func prettyTxIDError(resp *http.Response) error {
+	if resp.Status == "429 Too Many Requests"  {
```

Comment:
The preferred, compiler-checked code would be:
```!suggestion
	if resp.StatusCode == http.StatusTooManyRequests {
```
https://golang.org/pkg/net/http/#pkg-constants